### PR TITLE
pythonPackages.jupyterhub-ldapauthenticator: init at 1.1

### PR DESCRIPTION
**Pull request by ixxie**
- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    inherit pname version;
+    sha256 = "19dz3a3122wln8lkixj5jbh9x3cqlrcb3p7a53825cj72cmpcxwz";
+  };
+
+  # No tests in archive
+  doCheck = false;
+   
+  propagatedBuildInputs = [ jupyterhub ldap3 ];
+
+  disabled = pythonOlder "3.4";
```
Comment (dotlambda) [inline pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix]: Why that? The README says

> I've only tested with python3 - anyone willing to test with python2 is welcome to do so! There's no reason it shouldn't work.

File: pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+buildPythonPackage rec {
+  pname = "jupyterhub-ldapauthenticator";
+  version = "1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19dz3a3122wln8lkixj5jbh9x3cqlrcb3p7a53825cj72cmpcxwz";
+  };
+
+  # No tests in archive
```
Comment (dotlambda) [inline pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix]: There aren't even any on GitHub, so this should say something like `No tests implemented`

File: pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix [hunk 3]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (dotlambda) [inline pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix]: whitespace

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by ixxie** [comment]
@dotlambda  - whoopsy, been a little sloppy there! Fixed now, hopefully.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python3.pkgs.python3.pkgs.jupyterhub-ldapauthenticator

@ixxie You might want to add `disabled = !isPy3k`.

File: pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 = "19dz3a3122wln8lkixj5jbh9x3cqlrcb3p7a53825cj72cmpcxwz";
+  };
+
+  # No tests implemented
+  doCheck = false;
+   
+  propagatedBuildInputs = [ jupyterhub ldap3 ];
+
+  meta = with lib; {
+    description = "Simple LDAP Authenticator Plugin for JupyterHub.";
```
Comment (dotlambda) [inline pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix]: No `.`

File: pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, jupyterhub
+, ldap3
+, pythonOlder
```
Comment (dotlambda) [inline pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix]: Isn't used

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by ixxie** [comment]
@dotlambda - I applied the requested changes. I didn't use the `disabled = !isPy3k` because the github page for the package says this could in principle work on python2 but simply wasn't tested.

**Review by dotlambda — APPROVED** [review]

**Pull request merged**